client-src/clients/SockJSClient.js

```diff
@@ -4,7 +4,9 @@ const SockJS = require('sockjs-client/dist/sockjs');
 const { log } = require('../default/utils/log');
 const BaseClient = require('./BaseClient');
 
-module.exports = class SockJSClient extends BaseClient {
+module.exports = class SockJSClient extends (
```

Comment:
Cause by editor format?

Reply:
`prettier` was updated [to improve formatting for assignments of classes with long superclass names](https://prettier.io/blog/2020/11/20/2.2.0.html#improve-formatting-for-assignments-of-classes-with-long-superclass-names-9341httpsgithubcomprettierprettierpull9341-by-sosukesuzukihttpsgithubcomsosukesuzuki).

Reply:
Oh, thanks, weird for this to extra brackets, maybe fix by https://github.com/prettier/prettier/issues/9744

issue link https://github.com/prettier/prettier/pull/9341#issuecomment-732446575

Reply:
Is not important for this pr